docs/functionality/ipc_tinker.md

```diff
@@ -0,0 +1,47 @@
+# IPC slot value tinker
+
+Each ipc slot offers a dbus api interface.
+One for filters, get value and value tinkering.
+
+## General information
+The slot requests a name from dbus `<slot_name>._slot_`
+storing its dbus connection, 
+The implemented functionality inject their interfaces
+into the slots established dbus connection.
+
+### Interface names
+The following interface names are used for the functionality.
+
+- filters `<slot_name>/filters`
+- tinker  `<slot_name>/tinker`
+- value `<slot_name>/value`
+
+## Specification of the tinker interface
+A property called `forced` of the type bool,
+this property indicates weather the slots value is forced or not.
+
+A method called `force` accepting a value of the type of the slot,
+this method forces the value of the slot.
+
+A method called `release` accepting no arguments.
+Which releases the force on the slot having
```

Comment:
The idea is to "stick" a value.
So no, the next value should not override it.

Reply:
I think it would be safer to only implement a setter, you can set the value of the slot but it won't affect production as the next event will still be functional. I think it could be more harmful than good to block values from the signal.

So the use cases I have in mind for this feature is:
- IO testing inhouse manufacturing
- Service debugging 
- Optimizing pneumatic actuator timings
- Stop and start conveyors
- Change speed temporarily of a conveyor

I don't think any of those use cases require complete control of slot.

Do you have use cases in mind which would require blocking the following signal values?

Reply:
So this idea is modeled after
the plc world.

But, there we have a cyclic logic loop so many variables are overwritten in each cycle.

Here we only write on change so a setter is propably sufficient.

Reply:
@jbbjarnason I have updated the text to reflect this change.